templates/next/pages/foos/[id]/index.tsx

```diff
@@ -14,6 +14,14 @@ const Page: NextComponentType<NextPageContext, Props, Props> = ({ {{{lc}}} }) =>
       <div>
           <Head>
             <title>{`Show {{{ucf}}} ${ {{~lc}}['@id']}`}</title>
+            <script
+                type="application/ld+json"
+                dangerouslySetInnerHTML=\{{
+                  __html: JSON.stringify(
```

Comment:
Maybe could we also the text response coming from the API along with the decoded object in `dataAccess.ts` (https://github.com/api-platform/client-generator/blob/main/templates/next/utils/dataAccess.ts#L28) and inject the JSON directly? So we save a call to `JSON.stringify()`. WDYT?